index.js

```diff
@@ -256,6 +256,11 @@ module.exports = {
           "Float32Array",
           "Uint8Array",
           "Uint16Array",
+          "Int8Array",
+          "Uint8ClampedArray",
+          "Int16Array",
+          "Int32Array",
+          "Uint32Array",
```

Comment:
Don't be mad with me, but this list is supposed to be in alphabetic order. 😄

Reply:
will do. how about that "pc" on top?